packages/core/src/configuration/ConfigurationParser.ts

```diff
@@ -4,7 +4,11 @@ import { AeroGearConfiguration, ServiceConfiguration } from "./";
 /**
  * List of types of all supported services.
  */
-export type ServiceType = "metrics" | "keycloak" | "push" | any;
+export enum ServiceType {
+  METRICS = "metrics",
+  KEYCLOAK = "keycloak",
+  PUSH = "push"
```

Comment:
I don't think the core sdk need to know all the service types. It means we have to update this file every time when there is a new service is added. It also means when we change the name/type of a service module, the core module needs to be updated as well. Does it really needed?

I think the best place for those value should be in each service module itself.

Reply:
@wei-lee @josemigallas how about

Reply:
You're probably right, maintainability-wise this is not the right approach.

Reply:
yes, I think this is better.